content/en/platform/corda/5.2/developing-applications/cordapp-template/overview/_index.md

```diff
@@ -0,0 +1,101 @@
+
+### corda-runtime-plugin-cordapp
+
+| <div style="width:220px">Helper</div> | Description                                                                                                                                                                                  |
+| ------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `createGroupPolicy`                   | Creates the {{< tooltip >}}group policy{{< /tooltip >}} which is required to set up the {{< tooltip >}}application network{{< /tooltip >}}.                                                  |
+| `createKeyStore`                      | Creates the signing keys for publishing the {{< tooltip >}}CPIs{{< /tooltip >}}.                                                                                                             |
+| `buildCPIs`                           | Builds your CorDapp and wraps it in a signed CPI.                                                                                                                                            |
+| `deployCPIs`                          | Deploys the CPI to your local Corda cluster. If the CPI has already been deployed to the nodes on the cluster, this task performs a forced upload and replaces the old CPI with the new one. |
+
+Some tasks have a dependency on the others. So, for example, if you run `deployCPIs`, it also runs `createGroupPolicy`, `createKeyStore` and `buildCPIs`.
+
+### corda-runtime-plugin-network
+
+| <div style="width:220px">Helper</div> | Description                                                                                                                  |
+| ------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------- |
+| `vNodesSetup`                         | Sets up the virtual nodes specified in `config/static-network-config.json` on you local Corda cluster with the uploaded CPI. |
+
+Some tasks have a dependency on the others. So, for example, if you run `vNodesSetup`, it also runs `deployCPIs`.
+
+{{< note >}}
+You only need to run `vNodesSetup` the first time you upload your CPI to the corda cluster. On subsequent builds, you can just run `deployCPIs`. (`vNodeSetup` has no effect if the virtual nodes have already been set up. It will not try to recreate them.)
```

Comment:
```suggestion
You only need to run `vNodesSetup` the first time you upload your CPI to the Corda cluster. On subsequent builds, you can just run `deployCPIs`. (`vNodeSetup` has no effect if the virtual nodes have already been set up. It will not try to recreate them.)
```